plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConfig.java

```diff
@@ -152,4 +158,20 @@ public KafkaConfig setTimestampUpperBoundPushDownEnabled(boolean timestampUpperB
         this.timestampUpperBoundPushDownEnabled = timestampUpperBoundPushDownEnabled;
         return this;
     }
+
+    @NotNull
+    public List<@FileExists File> getResourceConfigFiles()
+    {
+        return resourceConfigFiles;
+    }
+
+    @Config("kafka.config.resources")
+    @ConfigDescription("Optional config files")
+    public KafkaConfig setResourceConfigFiles(List<String> files)
+    {
+        this.resourceConfigFiles = files.stream()
```

Comment:
Can you please validate if files exists and are readable? Please also test it.

Reply:
Hi @kokosing , is it necessary to validate that `files` exists here since we have defined above `@FileExists` annotation for the `get` method?

```java
    @NotNull
    public List<@FileExists File> getResourceConfigFiles()
    {
        return resourceConfigFiles;
    }
```

Reply:
Oh... thanks. Do you have tests that cover this? I think such tests could be added to `TestKafkaPlugin`

Reply:
Sorry for the delay. Sure I'll add one to cover this scenario.

Reply:
Added a new test.